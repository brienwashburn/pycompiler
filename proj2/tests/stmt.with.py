with foo: pass
with foo as bar: pass
with foo as bar, rax, quux as foo: 
    pass
