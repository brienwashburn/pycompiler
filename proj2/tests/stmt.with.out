(Module
 (With (((Name foo) #f)) (Pass))
 (With (((Name foo) (Name bar))) (Pass))
 (With
  (((Name foo) (Name bar)) ((Name rax) #f) ((Name quux) (Name foo)))
  (Pass)))
